experimental/bakeable/src/lib.rs

```diff
@@ -6,6 +6,14 @@ use quote::quote;
 #[cfg(feature = "derive")]
 pub use bakeable_derive::Bakeable;
 
+#[cfg(feature = "derive")]
```

Comment:
Both of these need to be always exported since it's possible to use `bakeable_derive` as a direct crate dependency.

Reply:
Can we make it so it's not?

Reply:
No, we can at best suggest that people do not do this

But the additional `quote` dependency is not a huge deal IMO.